src/main/java/com/Utils/CreatePropertiesFile.java

```diff
@@ -0,0 +1,40 @@
+package com.Utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class CreatePropertiesFile {
```

Comment:
think about what this class does what is responsible for and think about the name afterwards

Reply:
ReadPropertyClass?